package.json

```diff
@@ -80,6 +81,58 @@
         }
     },
     "contributes": {
+        "walkthroughs": [
+            {
+                "id": "jupyterWelcome",
+                "title": "Get started with Jupyter Notebooks",
+                "description": "Your first steps to set up a Jupyter project with all the powerful tools and features that the Jupyter extension has to offer!",
+                "when": "false",
+                "steps": [
+                    {
+                        "id": "jupyter.createNewNotebook",
+                        "title": "Create or open a Jupyter Notebook",
+                        "description": "Right click in the file explorer and create a new file with an .ipynb extension. Or, open the [Command Palette](command:workbench.action.showCommands) and run the command \n``Jupyter: Create New Blank Notebook``.\n[Create new Jupyter Notebook](command:toSide:jupyter.createnewnotebook)\n If you have an existing project, you can also [open a folder](command:workbench.action.files.openFolder) and/or clone a project from GitHub: [clone a Git repository](command:git.clone).",
+                        "media": {
+                            "image": "images/OpenOrCreateNotebook.gif",
```

Comment:
I think i remember VS Code adding support for themable images (svgs) for welcome pages.
Else someone on a light them (default) would get dark images.
@luabud might have more information about this.

Reply:
Also, do we need gifs for everything?
Python extension has SVGs and I personally prefer that, rather than just having dark images

Reply:
I'm a bit torn myself. The SVGs look much better, but at the same time the gifs feel like they explain things better.

Reply:
Agreed, but for some like opening a notebook, I don't think we need a gif, 
also thats the very first walkthrough, i'd like that to be themeble via svg

Reply:
Yea, I think Python walkthrough pages use them. I was thinking that once we start seeing positive numbers from the experiment, we'd ask Miguel to help us make them for us as well.

Reply:
I'm fine with that personally, you can track it how you want, but might be good to create an issue for that change of adding SVGs, just so it doesn't get forgotten in day to day shuffle..

Reply:
Agreed with Ian, approving

Reply:
FWIW there are SVGs that could be reused for this first tile, for example these: https://github.com/microsoft/vscode-python/blob/main/resources/walkthrough/create-notebook.svg or https://github.com/microsoft/vscode-python/blob/main/resources/walkthrough/data-science.svg